cmd/tke-installer/app/installer/installer.go

```diff
@@ -1284,33 +1283,6 @@ func (t *TKE) setupLocalRegistry(ctx context.Context) error {
 	return nil
 }
 
-func (t *TKE) startLocalRegistry() error {
```

Comment:
Where the images should be pushed to before tke-registry is installed?

Reply:
tke-registry create by nerdctl because nerdctl can not create containers inside container.